### DISCUSSION TITLE
Update to Spanish (Spain) translation [es-ES]

### DISCUSSION THREAD

**Discussion by miguelssm** [aclap-dev/video-downloadhelper#741] [Language & Translation]
[messages.json](https://github.com/aclap-dev/video-downloadhelper/files/15465712/messages.json)
This is my contribution.
Thanks for the development. Greetings from Spain

Miguel SSM

**Comment by paulrouget** [comment]
Hello. This is a translation file for Version 8 or legacy UI. We do not support those anymore. Please update your addon or use the new UI. Thank you.
